sites/NOAA_GFDL/default_gfdl.jsonc

```diff
@@ -22,14 +22,19 @@
       "CASE_ROOT_DIR" : "/archive/oar.gfdl.cmip6/ESM4/DECK/ESM4_historical_D1/gfdl.ncrc4-intel16-prod-openmp/pp",
       "pod_list": [
           // Optional: PODs to run for this model only (defaults to all)
-          "Wheeler_Kiladis",
-          "EOF_500hPa",
-         // "convective_transition_diag",
-          "MJO_suite",
-          "MJO_teleconnection",
+          //"Wheeler_Kiladis",
+          //"EOF_500hPa",
+          //"convective_transition_diag",
+          //"MJO_suite",
+          //"MJO_teleconnection",
           //"MJO_prop_amp",
-          "precip_diurnal_cycle"
-         // "SM_ET_coupling"
+          //"precip_diurnal_cycle"
+          //"SM_ET_coupling"
```

Comment:
Should all of the PODs be turned on?

Reply:
The new PODs have only been tested in an offline mode and we have seen several problems. Not sure if we should include them into the frepp framework at this time